feast_mariadb/Readme.md

```diff
@@ -29,47 +29,135 @@ d. You should see both databases deployments
 feast init -t postgres feast_mariadb
 ```
 
-### Step 3: Your `feature_store.yaml` file -- 
+### Step 3: Port-forward postgres and mariadb
+```
+oc port-forward pod/<pod_name> 5432
+Forwarding from 127.0.0.1:5432 -> 5432
+Forwarding from [::1]:5432 -> 5432
+```
+
+```
+oc port-forward pod/<pod_name> 33306:3306 -n feast
+Forwarding from 127.0.0.1:33306 -> 3306
+Forwarding from [::1]:33306 -> 3306
+```
+
+### Step 4: Your `feature_store.yaml` file for local deployment -- 
 ```
 project: feast_mariadb
-provider: aws
+provider: local
 registry:
     registry_type: sql
-    path: mysql://feast:feast@127.0.0.1:33306/feast
+    path: mysql://feast:feast@127.0.0.1:33306/sampledb
     cache_ttl_seconds: 60
 online_store:
     type: postgres
     host: localhost
     port: 5432
-    database: feast
-    db_schema: feast
+    database: sampledb
     user: feast
     password: feast
 offline_store:
     type: postgres
     host: localhost
     port: 5432
-    database: feast
-    db_schema: feast
+    database: sampledb
```

Comment:
Does it make sense to have the same database `sampledb` for both o line and offline store? (Just asking, with the goal of emulating a real world setup)

Reply:
My suspicion is that you'd be right in prod scenarios. That said, we'll be running some RH demos with shared DB's IIUC (thus the ask for Maria support) so good to know it works like this as well.